src/pages/questionPage.js

```diff
@@ -3,12 +3,14 @@
 import {
   ANSWERS_LIST_ID,
   NEXT_QUESTION_BUTTON_ID,
-  USER_INTERFACE_ID,
+  USER_INTERFACE_ID, QUESTION_NUMBER
 } from '../constants.js';
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
 
+export let questionProgress = 0;
```

Comment:
🙅‍♂️ Any We don't need another variable for this! Use what we have in `data.js`

Reply:
I created a variable in Data.js